packages/gatsby/src/internal-plugins/load-babel-config/gatsby-node.js

```diff
@@ -159,8 +159,6 @@ const addDefaultPluginsPresets = (
       loose: true,
       modules: false,
       useBuiltIns: `usage`,
-      sourceType: `unambiguous`,
-      shippedProposals: true, // includes async/await and Object spread/rest
```

Comment:
@existentialism mentioned also in the babel slack while I was figuring this out that beta 43/44 shipped support for ES2018 so this isn't needed anymore.